### PR TITLE
Allow stencil-start to run on case-sensitive file systems.

### PR DESCRIPTION
**Pull request by jnorth**
On case-sensitive file systems, the `stencil-start` script would cause an error when importing the Cycles module.

### CODE REVIEW

File: bin/stencil-start [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 var _ = require('lodash');
 var Bs = require('browser-sync').create();
-var Cycles = require('../lib/cycles');
+var Cycles = require('../lib/Cycles');
```
Comment (mickr) [inline bin/stencil-start]: Can you rename the file to `cycles.js` and test it, that will put it more within standards.

Comment (jnorth) [inline bin/stencil-start]: Good call 👍

**Comment by mickr** [comment]
👍

**Comment by hegrec** [comment]
👍

**Pull request merged**